themes/default/content/blog/mlops-huggingface-llm-aws-sagemaker-python/index.md

```diff
@@ -0,0 +1,131 @@
+---
+date: 2023-09-08
+title: "Deploy a Huggingface Model on AWS SageMaker with Pulumi Python IaC"
```

Comment:
Is it `Huggingface` (one word) or `Hugging Face` (two words)? I've seen it both ways. The template, for example, has it as two words.

Reply:
I looked it up when scott mentioned it during template review. The corp is `Hugging Face Inc.` however they are also routinely marketed as either the emoji or `Huggingface`. Given the corporate name I'm okay with committing to Hugging Face and we can make huggingface all together the meta tag.

Reply:
Fixed throughout in 46fb9a7